### PR TITLE
Changed name of generated documents in _generate_report.

### PR DESCRIPTION
**Pull request by ghost**
This change adds more information to doc's title.

### CODE REVIEW

**Comment by mkurek** [comment]
could you write (unit) test for it (mocking external service result)?

**Comment by szok** [comment]
:ok:

File: src/ralph/back_office/tests/test_models.py [hunk 1]
```diff
@@ -277,3 +281,17 @@ def test_return_report_when_user_not_assigned(self):
...
+        mock_method.return_value = GENERATED_FILE_CONTENT
+        report_template = ReportTemplateFactory(template__data=REPORT_TEMPLATE)
+        request = RequestFactory().get('/')  # only request's user is important
+        request.user = UserFactory()
+        instances = [BackOfficeAssetFactory(user=UserFactory())]
+
+        attachment = BackOfficeAsset._generate_report(
+            report_template.name, request, instances, report_template.language)
+
+        self.assertEqual(attachment.file.read(), GENERATED_FILE_CONTENT)
```
Comment (mkurek) [inline src/ralph/back_office/tests/test_models.py:307]: please validate here original name of the asset as well

**Comment by mkurek** [comment]
:ok:

**Pull request merged**